.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["next/core-web-vitals", "prettier"]
```

Comment:
This small change allows you to use prettier with nextjs pre-configured eslinting